docs/docsite/rst/contributor_guide.rst

```diff
@@ -0,0 +1,222 @@
+.. _ansible_collections.community.hashi_vault.docsite.contributor_guide:
+
+*****************
+Contributor guide
```

Comment:
Hm, the GitHub standard is to add a `contributing.md` file, right? Maybe add one and point to this file or to the built page?

Reply:
I will leave `contributing.md` for a later time, but I have updated the (previously blank) section in the README.